scripts/dataProvider.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * dataProvider.js
```

Comment:
I think this script should be called sales-center.js.

Reply:
@varunmitra mentioned that he wanted to be able to add more functions to the script for other templates, hence the generic name.